### ISSUE TITLE
"const map<EnumType, ?>" fails to parse

### ISSUE THREAD

**Issue by emilssolmanis** [eleme/thriftpy#75]
The thrift def

```thrift
enum Enum {
    A = 1
}

const map<Enum, string> ConstMap = {
  Enum.A: "hurr-durr"
}
```
fails to parse with

```
    def cast(self, data):
        dct = {}
        keys = data.keys()
    
        for key in keys:
>           dct[self[1].cast(key)] = self[2].cast(data[key])
E           AttributeError: 'str' object has no attribute 'cast'

../thriftpy/parser/model.py:141: AttributeError
```

**Comment by hit9** [comment]
Similar to issue https://github.com/eleme/thriftpy/pull/69.

**Comment by hit9** [comment]
Will be fixed in https://github.com/eleme/thriftpy/pull/80

**Comment by hit9** [comment]
Closed in #80
